api/python/t4/packages.py

```diff
@@ -527,6 +527,13 @@ def push(self, name, path):
             A new package that points to the copied objects
         """
         dest = fix_url(path)
+        if dest.startswith('file://'):
+            # local push
+            os.makedirs(path, exist_ok=True)
+            pkg = self._materialize(dest)
+            pkg.build(name)
```

Comment:
probably want to start sanitizing `name` on push/build/etc. to prevent directory traversal now